spec/attachment_spec.rb

```diff
@@ -0,0 +1,105 @@
+require "spec_helper.rb"
+
+@access_token = nil
+@cipher_uuid = nil
+@cipher = nil
+
+describe "attachment module" do
+  before do
+    FileUtils.rm_r "tmp/spec" if Dir.exist?("tmp/spec")
```

Comment:
Attachments aren't written to the filesystem anymore, so shouldn't all of this `tmp/spec` stuff be removed?

Reply:
Yes! I removed it along with the keep file for data/attachments. Thanks for catching this.